### ISSUE TITLE
Styles should preferably be controlled with CSS and not JS

### ISSUE THREAD

**Issue by ericluwj** [marcio/react-skylight#30]
By relying on JS to set inline styles, I am now finding it very troublesome to implement mobile friendliness. If the styles were controlled with CSS, I could have easily used media queries.

**Comment by marcio** [comment]
@ericluwj thanks for your feedback. I'm thinking an elegant way to use a separated CSS file, instead  inline JS styles, maybe an option to escape in line JS styles.

**Comment by AdamSkaidon** [comment]
👍 would very much like this over having it in the JS and inline. :)

**Comment by viridia** [comment]
I'd like a way to add an CSS class name to the main dialog element.

**Comment by matt16749** [comment]
:+1 any updates?

**Comment by jakefavour** [comment]
@viridia : yes being able to pass class names to the components would help a lot.
